README.md

```diff
@@ -1,2 +1,71 @@
-# SpotifyArtistPlaylistMaker
-Open Source project to create Spotify playlists with songs from selected artists
+# SpotifyArtistPlaylistMaker (working-title)
+> Create Spotify playlists with songs from selected artists
+
+![Help Wanted][help-shield]
+
+If sometimes you just feel like listening to songs from a couple of specific artists on random mode, then you're on the right place.
+
+<!-- ![](header.png) -->
+
+## Development setup
+
+SpotifyArtistPlaylistMaker was built using react and npm.
+
+1. Make sure you have [node](https://nodejs.org/en/) installed
+
+2. Clone the repository
+
+3. Install the project dependencies:
+```sh
+npm install
+```
+
+4. Run the project:
+```sh
+npm start
+```
+
+5. If everything goes according to plan you should see the app running on http://localhost:3000
+
```

Comment:
Add a linting topic;
Add `language-babel`;
Change default line ending to LF at `line-ending-selector`;